<launch>
	
	<node name="pose_command_action_server" pkg="cubic_polynomial_planner" type="pose_command_action_server" output="screen"/>
	
</launch>